test/e2e/install.go

```diff
@@ -59,9 +59,12 @@ func veleroInstall(ctx context.Context, cli, veleroImage string, resticHelperIma
 			return errors.New("No object store provider specified - must be specified when using kind as the cloud provider") // Gotta have an object store provider
 		}
 	}
-
+	version, err := getVeleroVersion(ctx, cli, true)
+	if err != nil {
+		return errors.WithMessage(err, "Failed to get velero version")
+	}
 	// Fetch the plugins for the provider before checking for the object store provider below.
-	providerPlugins := getProviderPlugins(objectStoreProvider)
+	providerPlugins := getProviderPlugins(version, objectStoreProvider)
```

Comment:
Use the user provided `plugins` first, if it isn't provided, then call `getProviderPlugins`